contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -263,7 +287,7 @@ contract SubmissionProxy is Ownable {
                     revert IndexOutOfBounds();
                 }
 
-                if (isWhitelisted(signer_) && (signer_ != oracles[oracleIndex_])) {
+                if (isWhitelisted(signer_) && (signer_ == oracles[oracleIndex_])) {
```

Comment:
I'm not sure if it is necessary to check `signer_ == oracles[oracleIndex_]` as `oracleIndex_` is declared from `whitelist[signer_].index` I can't imagine where it could be different. It is auto set from the code on adding oracle

Reply:
updated at https://github.com/Bisonai/orakl/pull/1380/commits/4b1d88472a401160aa7d559e6f6ae2552cc068f5